src/OrchardCore.Modules/OrchardCore.Lucene/Startup.cs

```diff
@@ -86,6 +86,8 @@ public override void ConfigureServices(IServiceCollection services)
 
         public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder routes, IServiceProvider serviceProvider)
         {
+            serviceProvider.GetRequiredService<LuceneIndexingService>().InitializeAsync().GetAwaiter().GetResult();
```

Comment:
Should we do this in a `ModularTenantEvents` `ActivatingAsync` method, rather than direct in startup?

Reply:
Question is should it try to self heal every time we activate the Lucene module or only once on the app startup.
I like the idea.

Reply:
It bites it's own tail unfortunately.

Reply:
Done